doc/clusters/cadence.md

```diff
@@ -142,25 +156,54 @@ metadata:
   name: cadence-sample
 spec:
   name: "CadenceCluster"
-  version: "0.24.0"
+  version: "1.0.0"
   standardProvisioning:
     - targetCassandra:
-        dependencyCdcId: "66abf438-0020-4e2d-a39b-78706b7de064"
+        dependencyCdcId: "9d43ac54-7317-4ce5-859a-e9d0443508a4"
         dependencyVpcType: "VPC_PEERED"
+  packagedProvisioning:
+    - bundledCassandraSpec:
+        nodeSize: "CAS-DEV-t4g.small-5"
+        network: "10.2.0.0/16"
+        replicationFactor: 3
+        nodesNumber: 3
+        privateIPBroadcastForDiscovery: false
+        passwordAndUserAuth: true
+        useAdvancedVisibility: true
+        bundledKafkaSpec:
+          nodeSize: "KFK-DEV-t4g.small-5"
+          nodesNumber: 3
+          network: "10.3.0.0/16"
+          replicationFactor: 3
+          partitionsNumber: 3
+        bundledOpenSearchSpec:
+          nodeSize: "SRH-DEV-t4g.small-5"
+          replicationFactor: 3
+          network: "10.4.0.0/16"
+  twoFactorDelete:
+    - email: "example@netapp.com"
   privateNetworkCluster: false
   dataCentres:
-    - region: "US_EAST_1"
-      network: "10.3.0.0/16"
+    - region: "US_EAST_2"
+      network: "10.12.0.0/16"
+      # if you use multi-region mode please provide
+      # non-overlapping CIDR block for the secondary mode cluster
```

Comment:
In a multi-region mode setup, ensure the CIDR block for the secondary cluster does not overlap with the primary one

Reply:
fixed